pkg/kritis/crd/buildpolicy/buildpolicy.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package buildpolicy
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/grafeas/kritis/pkg/kritis/apis/kritis/v1beta1"
+	clientset "github.com/grafeas/kritis/pkg/kritis/client/clientset/versioned"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+)
+
+// ValidateFunc defines the type for Validating Build Policies
+type ValidateFunc func(bp v1beta1.BuildPolicy, buildFrom string) error
+
+// SigningPolicies returns all ISP's in the specified namespaces
+// Pass in an empty string to get all ISPs in all namespaces
+func BuildPolicies(namespace string) ([]v1beta1.BuildPolicy, error) {
+	config, err := rest.InClusterConfig()
```

Comment:
We should use one way to connect to the cluster.
We right now use this -> https://github.com/grafeas/kritis/blob/master/pkg/kritis/kubernetes/client.go#L31
A Todo for us to see which one is better.

Reply:
Not sure I follow, this is following the example in the security policy.